Lesson-7/lesson-7-new/index.html

```diff
@@ -0,0 +1,101 @@
+        <p class="attractions"> <a href="">Attractions</a> </p>
+        <img class="night_img" src="LinkedSprite.svg#moon" alt="night">
+        <img class="account" src="LinkedSprite.svg#account" alt="accountcircle">
+    </div>
+    <div class="container_middle">
+        <p class="heading">Discover stays to live, work or just relax</p>
+        <div class="label">
+            <label id="search" for="search" class="search">Your destination or hotel name</label>
+            <label id="date" for="date">Check-in — Check-out</label>
+        </div>
+        <div class="place_to_request">
+            <input type="search" name="search_bar" placeholder="New York">
+            <input class="calendar" type="date" name="calendar" placeholder="Check-in — Check-out">
+            <input type="text" name="text" placeholder="Adults — Children — Room">    
+            <input class="button" type="submit" value="Search" placeholder="Search">
+        </div>
+        <div class="phone_app">
+        <img class="icon_google" src="image/google-play-badge.png" alt="google">
+        <img class="icon_app_store" src="image/Download_on_the_App_Store_Badge_US-UK_RGB_blk_092917 1.png" alt="app">
+        </div>
+    </div>
+    </article>
+<article class="Wat_do_we_offer">
+    <h1 class="headline">What do we offer</h1>
+    <hr>
+    <div class="all_containers">
+    <div class="container_headphones">
+<img src="LinkedSprite.svg#headphones" alt="headphones" class="headphones">
+<p class="text_headphones">Support 24/7</p> 
+</div>
+<div class="container_communicate">
+<img src="LinkedSprite.svg#communicate" alt="communicate" class="communicate">
+<p class="text_communicate">Communicate directly</p> 
+</div>
+<div class="container_gadgets">
+<img src="LinkedSprite.svg#gadgets" alt="gadgets"  class="gadgets">
+<p class="text_gadgets">Book online</p> 
+</div>
+<div class="container_like">
+<img src="LinkedSprite.svg#like" alt="like" class="like_img">
+<p class="text_like">Real guest reviews</p>
+</div>
+</div>
+</article>
+<article class="last">
+</div>
+<div class="heading_logo">
+    <img src="image/logo_vector.png" alt="heading_logo">
+</div>
+<div class="container">
+<link rel="stylesheet" href="footer.css">
+<div class="container_about">
+    <ul>
+    <li class="heading_about">About</li>
+    <li>Now Triphouse works</li>
+    <li>Careers</li>
+    <li>Privacy</li>
+    <li>Terms</li>
+</ul>
+</div>
+<div class="container_property_types">
+    <ul>
+    <li class="heading_property">Property types</li>
+    <li>Guest houses</li>
+    <li>Hotels</li>
+    <li>Apartments</li>
+    <li>Villas</li>
+    <li>Holiday homes</li>
+    <li>Hostels</li>
+</ul>
+</div>
+<div class="container_support">
+    <ul>
+    <li class="heading_support">Support</li>
+    <li>Contact Customer Service</li>
+    <li>FAQ</li>
+</ul>
+</div>
+</div>
+<footer>
+    <p class="caption">&copy; 2020 Triphouse, lnc. All right reserved</p>
+</footer>
```

Comment:
for middle displays it is not showing correctly
![image](https://user-images.githubusercontent.com/20189313/128646168-ed18dcc3-0436-4c0d-ba0b-a1619caa6df3.png)